ush/templates/FV3LAM_wflow.xml

```diff
@@ -2361,6 +2318,54 @@ MODULES_RUN_TASK_FP script.
 {%- endif %}
 {%- endif %}
 
+{%- if do_recenter %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&RUN_RECENTER_TN;" cycledefs="recentercyc" maxtries="{{ maxtries_recenter }}">
+    &RSRV_ENKF;
+    &WALL_LIMIT_RECENTER;
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&RUN_RECENTER_TN;" &JOBSDIR;/JREGIONAL_RUN_RECENTER</command>
+    &RESOURCES_RUN_RECENTER;
+    &NATIVE_ALL;
+    <walltime>&WALLTIME_RUN_RECENTER;</walltime>
+    &NODESIZE_ALL;
+    <jobname><cyclestr>&TAG;_&RUN_RECENTER_TN;</cyclestr></jobname>
+    <join><cyclestr>&LOGDIR;/&RUN_RECENTER_TN;_@Y@m@d@H.log</cyclestr></join>
+    <envar><name>GLOBAL_VAR_DEFNS_FP</name><value>&GLOBAL_VAR_DEFNS_FP;</value></envar>
+
+    <envar><name>nens</name><value>{{ num_ens_members }}</value></envar>
+    <envar><name>CDATE</name><value><cyclestr>@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_DIR</name><value><cyclestr>&CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>CYCLE_ROOT</name><value><cyclestr>&CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>ENSCTRL_CYCLE_DIR</name><value><cyclestr>&ENSCTRL_CYCLE_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+    <envar><name>ENSCTRL_CYCLE_ROOT</name><value><cyclestr>&ENSCTRL_CYCLE_BASEDIR;</cyclestr></value></envar>
+    <envar><name>NWGES_DIR</name><value><cyclestr>&NWGES_BASEDIR;/@Y@m@d@H</cyclestr></value></envar>
+
+    <dependency>
+      <and>
+        {%- if do_nonvar_cldanal %}
+        <and>
+        {%- for m in range(1, num_ens_members+1) %}
+          <taskdep task="&CLDANL_NONVAR_TN;_prod_mem{{ "%04d" % m }}"/>
+        {%- endfor %}
+        </and>
+        {%- elif do_enkf_radar_ref %}
+        <taskdep task="&ENKF_RADAR_REF_TN;"/>
+        {%- elif do_enkfupdate %}
+        <taskdep task="&RUN_ENKFUPDT_TN;"/>
+        {%- endif %}
+        <or>
+        <datadep age="00:00:00:05"><cyclestr>&ENSCTRL_CYCLE_BASEDIR;/@Y@m@d@H/nonvar_cldanl/nonvarcldanl_complete.txt</cyclestr></datadep>
+        <datadep age="00:00:00:05"><cyclestr>&ENSCTRL_CYCLE_BASEDIR;/@Y@m@d@H/nonvar_cldanl_spinup/nonvarcldanl_complete.txt</cyclestr></datadep>
+        </or>
+      </and>
+    </dependency>
+  </task>
+
+{%- endif %}
+
```

Comment:
Just curious, Is the moving of the recenter task from line 1987 to 2321 purposely?

Reply:
Yes, it should go after cloud analysis, to make the workflow more readable.